cluster/selfplay/Makefile

```diff
@@ -31,6 +31,16 @@ cpu-image-clean:
 		rm staging/requirements.txt
 		rm staging/player_wrapper.sh
 
+cc-image:
+		mkdir -p staging/cc
+		cp ../../requirements.txt staging/
+		cp ../../WORKSPACE staging/
+		cp cc_wrapper.sh staging/
+		cp -r ../../cc/* staging/cc/
```

Comment:
these can be joined to a smaller number of lines if you want

cp ../../requirements.txt ../../WORKSPACE cc_wrapper.sh staging/